### PR TITLE
Ong and RCG Rhetoric for t-pen.org

### PR DESCRIPTION
**Pull request by thehabes**
Swap out or combine the mentions of Digital Humanities and Research Computing Group in page text.

### CODE REVIEW

**Comment by thehabes** [comment]
image

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # TPEN28
 ![](http://t-pen.org/TPEN/images/tpen_logo_header.jpg)
 
-The Transcription for Paleographical and Editorial Notation (T‑PEN) project is coordinated by the Center for Digital Theology at Saint Louis University (SLU) and funded by the Andrew W. Mellon Foundation and the NEH. The Electronic Norman Anonymous Project developed several abilities at the core of this project's functionality.
+The Transcription for Paleographical and Editorial Notation (T‑PEN) project was coordinated by the Center for Digital Theology at Saint Louis University (SLU) and funded by the Andrew W. Mellon Foundation and the NEH. The Electronic Norman Anonymous Project developed several abilities at the core of this project's functionality.
+
+T-PEN is currently (2022) maintained by the Research Computing Group at Saint Louis University.  T-PEN is no longer funded.
```
Comment (cubap) [inline README.md:6]: It is funded, but not by a monolithic program. I think we can drop the date and just say it is maintained by us.

File: web/about.jsp [hunk 2]
```diff
@@ -34,9 +34,9 @@
...
                             <p>T&#8209;PEN is released under <a href="http://www.opensource.org/licenses/ecl2.php" title="Educational Community License" target="_blank">ECL v.2.0</a> as free and open-source software (<a href="https://github.com/jginther/T-PEN/tree/master/trunk" target="_blank">git</a>), the primary instance of which is maintained by SLU at <a href="www.T&#8209;PEN.org" target="_blank">T&#8209;PEN.org</a>.
-                            </p>
+                            <p>T&#8209;PEN is currently (2022) maintained by the Research Computing Group at Saint Louis University and is no longer funded.</p>
```
Comment (cubap) [inline web/about.jsp:39]: as above

File: web/about.jsp [hunk 3]
```diff
@@ -110,6 +110,15 @@
                                 T&#8209;PEN as a web-based application.</p>
                             <p>  </p>
                         </li>
+                        <li class="gui-tab-section">
+                            <h3>Support/Maintenance Team <br> T-PEN 2.8-3.0</h3>
+                            <dl>                                
+                                <dt>Patrick Cuba, I.T. Architect</dt>
```
Comment (cubap) [inline web/about.jsp:116]: tehnically... I'm  "IT Architect" -  I dunno why

File: web/about.jsp [hunk 4]
```diff
@@ -144,8 +153,8 @@
                                 <dd>University&nbsp;of&nbsp;Kentucky</dd>
                                 <dt>Jon Deering, Senior Developer</dt>
                                 <dd>Center&nbsp;for&nbsp;Digital&nbsp;Theology, Saint&nbsp;Louis&nbsp;University</dd>
-                                <dt>Patrick Cuba, Web Developer</dt>
```
Comment (cubap) [inline web/about.jsp:147]: I can be left with my old title for this period, I think.

**Review by cubap — CHANGES_REQUESTED** [review]

**Pull request merged**